assemblies/features/base/src/main/filtered-resources/resources/bin/karaf

```diff
@@ -322,6 +322,7 @@ run() {
                     --add-exports=jdk.naming.rmi/com.sun.jndi.url.rmi=ALL-UNNAMED \
                     --add-exports=java.rmi/sun.rmi.registry=ALL-UNNAMED \
                     --add-exports=java.security.sasl/com.sun.security.sasl=ALL-UNNAMED \
+                    --add-exports=java.naming/com.sun.jndi.ldap=ALL-UNNAMED \
```

Comment:
This change should be added to the wrapper configuration as well. Can you please include it ? I can do it for you if you prefer.

Reply:
Added this to unix wrapper configuration.

Reply:
added to windows and windows64 wrappers also + karaf.bat

Reply:
Awesome, thanks ! I will review, squash and merge.